frontend/src/app/modules/home/projects/create-project/create-project.component.ts

```diff
@@ -0,0 +1,180 @@
+            { label: 'Server', command: (event) => this.onTabChange(event) },
+            { label: 'Mobile', command: (event) => this.onTabChange(event) },
+            { label: 'Desktop', command: (event) => this.onTabChange(event) }
+        ];
+        this.activeTabItem = Object.assign(this.tabItems[0]);
+        this.onTabChange();
+        this.selectedAlertType = Object.assign(this.alertTypes[0]);
+        this.selectedAlertTimeInterval = Object.assign(this.alertTimeIntervals[0]);
+
+        this.formGroup = new FormGroup({
+            alertCategory: new FormControl(
+                this.alertCategories[0],
+                [
+                    Validators.required
+                ]
+            ),
+            projectName: new FormControl(
+                '',
+                [
+                    Validators.required,
+                    Validators.minLength(3),
+                    Validators.maxLength(50),
+                    Validators.pattern('^[a-zA-Z0-9-_]+$')
+                ]
+            ),
+            team: new FormControl(
+                this.userTeams[0],
+                [
+                    Validators.required
+                ],
+            ),
+        });
+    }
+
+    onTabChange(event?: any): void {
+        if (this.selectedPlatformElement) {
+            this.selectedPlatformElement.element.style.background = 'transparent';
+            this.selectedPlatformElement = undefined;
+        }
+        switch (event?.item.label ?? '') {
+            case 'Browser': {
+                this.viewPlatformCards = [...this.platformCards.filter(value => value.isBrowser === true)];
+                break;
+            }
+            case 'Server': {
+                this.viewPlatformCards = [...this.platformCards.filter(value => value.isServer === true)];
+                break;
+            }
+            case 'Mobile': {
+                this.viewPlatformCards = [...this.platformCards.filter(value => value.isMobile === true)];
+                break;
+            }
+            case 'Desktop': {
+                this.viewPlatformCards = [...this.platformCards.filter(value => value.isDesktop === true)];
+                break;
+            }
+            default: {
+                this.viewPlatformCards = [...this.platformCards];
+                break;
+            }
+        }
+    }
+
+    onPlatformSelected(event: any, platformName: string) {
+        if (this.selectedPlatformElement) {
+            this.selectedPlatformElement.element.style.background = 'transparent';
+            if (this.selectedPlatformElement.platformName === platformName) {
+                this.selectedPlatformElement = undefined;
+                return;
+            }
+        }
+        const element = event.target.className !== 'card ng-star-inserted' ? event.target.parentElement : event.target;
+        element.style.background = 'lightgray';
+        this.selectedPlatformElement = { element, platformName };
```

Comment:
try to avoid working with DOM elements. Built-in Angular directives should help you with that

Reply:
![image](https://user-images.githubusercontent.com/36418695/128047739-54a0baad-b8a0-4aed-8e54-5dd0cbdff491.png)

I had no idea how to implement the platform pick. So i have implemented something like you see.
Yeah, i will try to make this without DOM, but i think it would be hard :D

Reply:
![image](https://user-images.githubusercontent.com/36418695/128060322-e42010f1-edcd-4e59-925f-d2e85a8092f1.png)
![image](https://user-images.githubusercontent.com/36418695/128060364-62b7cbcc-1596-4313-92a1-196fa8daa307.png)
![image](https://user-images.githubusercontent.com/36418695/128060389-74745000-6cb0-42dc-9db3-b28fac069919.png)
Something like this?

Reply:
Implemented built-in angular directive `[ngClass]`